scripts/slider.js

```diff
@@ -26,4 +26,8 @@ function changeSlide() {
     $("#slider").fadeIn(500);
     timer1 = setTimeout("changeSlide()", 5000);
     timer2 = setTimeout("hide()", 4500);
-}
\ No newline at end of file
+}
+
+$(document).ready(function changeSlide(){
```

Comment:
It seems for me that you misuse callback here.
It may work, but it is not ideal.
You should either put changeSlide as a callback to ready function:
$(document).ready(changeSlide);
or provide it as a function call in anononymous function as a callback:
$(document).ready(function (){
  changeSlide();
});
Please check all options.
If all of them work, then you may consider choosing the first one as something least complicated.